backend/Origam.Gui.Win/Commands/ShowDataStructureSql.cs

```diff
@@ -0,0 +1,86 @@
+#region license
+/*
+Copyright 2005 - 2021 Advantage Solutions, s. r. o.
+
+This file is part of ORIGAM (http://www.origam.org).
+
+ORIGAM is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+ORIGAM is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with ORIGAM. If not, see <http://www.gnu.org/licenses/>.
+*/
+#endregion
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using Origam.DA.Service;
+using Origam.Schema.EntityModel;
+using Origam.UI;
+using Origam.Workbench.Services;
+using Origam.Workbench.Services.CoreServices;
+
+namespace Origam.Gui.Win.Commands;
+
+public class ShowDataStructureSql : AbstractMenuCommand
+{
+    private WorkbenchSchemaService _schema =
+        ServiceManager.Services.GetService(typeof(SchemaService)) as WorkbenchSchemaService;
+    public override bool IsEnabled
+    {
+        get { return Owner is DataStructure; }
+        set { base.IsEnabled = value; }
+    }
+
+    public override void Run()
+    {
+        AbstractSqlDataService abstractSqlDataService =
+            DataServiceFactory.GetDataService() as AbstractSqlDataService;
+        AbstractSqlCommandGenerator generator = (AbstractSqlCommandGenerator)
+            abstractSqlDataService.DbDataAdapterFactory.Clone();
+        DataStructure ds = Owner as DataStructure;
```

Comment:
We don't usually use single/two character variable names. In this case `dataStructure` would be preferered.